### ISSUE TITLE
Change 'Next week' date option to '+7 days'

### ISSUE THREAD

**Issue by ngm** [orgzly-revived/orgzly-android-revived#318]
When setting a date (for scheduled or deadline), you have three options for quickly choosing a date: 

- Today
- Tomorrow
- Next week

It seems to be that the purpose of these is to say it's urgent  so I'll do it today, it's not urgent but it's important so I'll do it tomorrow, or it's not urgent or important so I'll do it in a week from now. 

This is good, except:

Next week sets the date to the following Monday. It does this regardless of whatever the current day is. 

So if today is Sunday, choosing tomorrow or next week amounts to the same thing. 

I think it would be better if 'Next week' was replaced with '+7 days'. So really those options mean +0, +1, or +7.

**Comment by ngm** [comment]
It could just be in my way of doing task management that +7 makes more sense. Others might prefer 'next week' always meaning next Monday. (I'd be interested to hear why).

In which case, perhaps it could be a config option as to what that button does.

**Comment by alensiljak** [comment]
But isn't +7 as arbitrary as any other number?

**Comment by ngm** [comment]
Not totally arbitrary, as there's 7 days in a week.  So it means a 'week from now'. +5 or +9 would be more arbitrary.

**Comment by alensiljak** [comment]
True. But then the title would be `in 7 days`, not `next week`. Which would be OK option if that's what you're used to. However, be careful with suggesting changing existing defaults as many people would be negatively surprised by the changes.
Adding features is generally fine as that does not modify long-term system's behaviour for existing users.
That was the point of my question.
Creating an option to add 7, 9, 15, or any n days is fine. But switching "next week" to mean something completely different is a bit weird. There is nothing wrong or right with "next week" meaning "the next Monday". But years of it meaning one specific thing give it a lot of weight.
Unfortunately, we don't have any statistics to see which features are used by the users, how, and how much, so we can't say which of them are (un)popular. But switching established defaults is usually a bad idea.

**Comment by ngm** [comment]
Fair enough, agreed that changing a long-standing behaviour can be problematic.
